### PR TITLE
fix(nemesis.py): Surround elasticsearch publishing with a try clause

### PR DESCRIPTION
**Pull request by ShlomiBalalis**
When sct publishes data to elastic search, there's the possibility that
an exception will occur (namely a timeout). When such error takes place
the nemesis thread will fail, and no new nemeses will start in this run
(see fde13648-baf4-4e37-b401-97e5a2d658f2).
Therefore, I surrounded the elastic search publishing in sct with a try clause.

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by roydahan** [comment]
@bentsi to approve and merge.

File: sdcm/nemesis.py [hunk 1]
```diff
@@ -2990,7 +2991,12 @@ def wrapper(*args, **kwargs):  # pylint: disable=too-many-statements
             disrupt = args[0].get_disrupt_name()
             del log_info['operation']
 
-            args[0].update_stats(disrupt, status, log_info)
+            try:  # So that the nemesis thread won't stop due to elasticsearch failure
+                args[0].update_stats(disrupt, status, log_info)
+            except ElasticSearchConnectionTimeout:
```
Comment (fgelcer) [inline sdcm/nemesis.py:2996]: maybe you can after your message add the exception itself?
like:
```
except ElasticSearchConnectionTimeout as ex:
    args[0].log.warning("Connection timed out when attempting to update elasticsearch statistics")
    args[0].log.warning(ex)
```

File: sdcm/nemesis.py [hunk 2]
```diff
@@ -2990,7 +2991,12 @@ def wrapper(*args, **kwargs):  # pylint: disable=too-many-statements
...
+                args[0].log.warning("Connection timed out when attempting to update elasticsearch statistics")
+            except:
```
Comment (fgelcer) [inline sdcm/nemesis.py:2998]: same here

**Comment by ShlomiBalalis** [comment]
@bentsi Any input?

**Review by fgelcer — APPROVED** [review]
LGTM

**Review by bentsi — APPROVED** [review]

**Pull request merged**